api/groups/eventsGroup_test.go

```diff
@@ -109,18 +98,10 @@ func TestEventsGroup_PushEvents(t *testing.T) {
 
 		jsonBytes, _ := json.Marshal(argsSaveBlockData)
 
-		wasCalled := false
-		wasCalledV2 := false
-
 		args := createMockEventsGroupArgs()
-		args.Facade = &mocks.FacadeStub{
-			HandlePushEventsV1Called: func(events data.SaveBlockData) error {
-				wasCalled = true
-				return errors.New("facade error")
-			},
-			HandlePushEventsV2Called: func(events data.ArgsSaveBlockData) error {
-				wasCalledV2 = true
-				return errors.New("facade error")
+		args.PayloadHandler = &testscommon.PayloadHandlerStub{
+			ProcessPayloadCalled: func(payload []byte, topic string) error {
+				return errors.New("expected err")
```

Comment:
Can we somehow check that this error was returned?

Reply:
Or at least just check the old `wasCalled` variable

Reply:
added was called check in all tests